src/world/World.php

```diff
@@ -588,6 +603,140 @@ private function initRandomTickBlocksFromConfig(ServerConfigGroup $cfg) : void{
 		}
 	}
 
+	public function getWeather() : WeatherType{
+		return $this->weather;
+	}
```

Comment:
As I said previously, "weather type" should be confined to commands only. It doesn't make sense anywhere else considering the dynamic rain & thunder strengths.